var/spack/repos/builtin/packages/blaspp/package.py

```diff
@@ -4,90 +4,49 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
+import os
 
 
 class Blaspp(CMakePackage, CudaPackage):
     """C++ API for the Basic Linear Algebra Subroutines. Developed by the
        Innovative Computing Laboratory at the University of Tennessee,
        Knoxville."""
 
-    homepage = "https://bitbucket.org/icl/blaspp"
-    git      = "https://bitbucket.org/icl/blaspp"
+    homepage = 'https://bitbucket.org/icl/blaspp'
+    git = homepage
+    url = 'https://bitbucket.org/icl/blaspp/downloads/blaspp-2020.09.00.tar.gz'
     maintainers = ['teonnik', 'Sely85', 'G-Ragghianti', 'mgates3']
 
-    version('develop', commit='6293d96')
+    version('master', branch='master')
+    version('2020.10.00', sha256='ce148cfe397428d507c72d7d9eba5e9d3f55ad4cd842e6e873c670183dcb7795')
+    version('2020.09.00', sha256='ee5d29171bbed515734007dd121ce2e733e2f83920c4d5ede046e657f4a513ef')
 
-    variant('gfort',
-            default=False,
-            description=('Use GNU Fortran interface. '
-                         'Default is Intel interface. (MKL)'))
-    variant('ilp64',
-            default=False,
-            description=('Use 64bit integer interface. '
-                         'Default is 32bit. (MKL & ESSL)'))
-    variant('openmp',
-            default=False,
-            description=('Use OpenMP threaded backend. '
-                         'Default is sequential. (MKL & ESSL)'))
+    variant('openmp', default=True, description='Use OpenMP internally.')
+    variant('cuda',   default=True, description='Build with CUDA')
```

Comment:
Using `cuda=True` by default is a bit untypical for packages in Spack.

Also, since CUDA is not supported on macOS anymore, this means that the default call of `spack install blaspp` will just error out for all macOS users.

Reply:
> setting spack install blaspp -cuda does not deactivate cuda if the variant line is added this way,

I don't find this to be the case.  Can you elaborate?

```
$ spack spec blaspp -cuda ^openblas@0.3.5
Input spec
--------------------------------
blaspp~cuda
    ^openblas@0.3.5

Concretized
--------------------------------
blaspp@2020.10.02%gcc@7.4.0~cuda~ipo+openmp+shared build_type=RelWithDebInfo cuda_arch=none arch=linux-centos7-skylake_avx512
    ^cmake@3.19.2%gcc@7.4.0~doc+ncurses+openssl+ownlibs~qt arch=linux-centos7-skylake_avx512
        ^ncurses@6.2%gcc@7.4.0~symlinks+termlib arch=linux-centos7-skylake_avx512
            ^pkgconf@1.7.3%gcc@7.4.0 arch=linux-centos7-skylake_avx512
        ^openssl@1.1.1i%gcc@7.4.0+systemcerts arch=linux-centos7-skylake_avx512
            ^perl@5.32.0%gcc@7.4.0+cpanm+shared+threads arch=linux-centos7-skylake_avx512
                ^berkeley-db@18.1.40%gcc@7.4.0 arch=linux-centos7-skylake_avx512
                ^gdbm@1.18.1%gcc@7.4.0 arch=linux-centos7-skylake_avx512
                    ^readline@8.0%gcc@7.4.0 arch=linux-centos7-skylake_avx512
            ^zlib@1.2.11%gcc@7.4.0+optimize+pic+shared arch=linux-centos7-skylake_avx512
    ^openblas@0.3.5%gcc@7.4.0~consistent_fpcsr~ilp64+pic+shared threads=none arch=linux-centos7-skylake_avx512
```

Reply:
Oh, the last case (always requiring cuda) might be an issue only on macOS that a user reported to me.
I don't have that problem on macOS. Will investigate on this more!

Reply:
I'm good with switching the CUDA default to False. That will make more sense as we add ROCm and Intel support.